hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/SQLDelegationTokenSecretManager.java

```diff
@@ -415,6 +458,8 @@ public int incrementCurrentKeyId() {
   // Token operations in SQL database
   protected abstract byte[] selectTokenInfo(int sequenceNum, byte[] tokenIdentifier)
       throws SQLException;
+  protected abstract Map<byte[], byte[]> selectTokenInfos(long maxModifiedTime, int topResults)
```

Comment:
Consider renaming to `selectUnrenewedTokenInfos` of something that indicates the subset of tokens we are fetching.

Reply:
I thought that having the maxModifiedTime in the parameter list would make it clear enough, but I'll still rename the method to prevent confusion